examples/streaming/index.tsx

```diff
@@ -122,14 +123,19 @@ async function runStreamingExample() {
 
   console.log("\n🚀 Starting streaming example with prompt:", prompt);
 
-  const workflow = gsx.Workflow("StreamStoryWorkflow", StreamStory);
+  const workflow = gsx.Workflow("StreamStoryWorkflow", StreamStory, {
```

Comment:
It can be set as either an opt when creating the workflow, or when calling `workflow.run`.